rke-templates/3-node-certificate-recognizedca.yml

```diff
@@ -116,6 +116,25 @@ addons: |-
           - --no-cacerts
           imagePullPolicy: Always
           name: cattle-server
+  #       env:
+  #       - name: HTTP_PROXY
+  #         value: "http://your_proxy_address:port"
+  #       - name: HTTPS_PROXY
+  #         value: "http://your_proxy_address:port"
+  #       - name: NO_PROXY
+  #         value: "localhost,127.0.0.1,0.0.0.0,10.43.0.0/16,your_network_ranges_that_dont_need_proxy_to_access"
+          livenessProbe:
+            httpGet:
+              path: /ping
+              port: 80
+            initialDelaySeconds: 60
```

Comment:
@superseb are the delay/period parameters arbitrary, or determined based on the testing you've done?

Reply:
It's largely dependent on the health of the machines (responsiveness/network/disk), but with an intialdelay of 60 seconds for the livenessprobe and additional 3 * 60 before it's restarted, sounds like enough room.

readinessprobe will only put the pod in unready state, we can start checking earlier. Normal startup for container is about 20 seconds, and it will retry and become ready after max 10 seconds.